src/components/form/dt-form-base.js

```diff
@@ -109,6 +109,10 @@ export default class DtFormBase extends LitElement {
   }
 
   labelTemplate() {
+    if (!this.label) {
+      return '';
```

Comment:
@incraigulous Tests are failing because this is a normal string instead of html``. It would probably be easier to just return null.

Reply:
@cairocoder01 Returning null didn't work. I had to go into each test that was expecting a label and add a label to the fixture.

Reply:
We need to make sure that the component won't throw errors like that if a label isn't given. You probably need to use that `html` function to return an empty string:
```
return html``;
```